src/ui_interface.h

```diff
@@ -122,6 +122,7 @@ void InitWarning(const bilingual_str& str);
 
 /** Show error message **/
 bool InitError(const bilingual_str& str);
+constexpr auto AbortError = InitError;
```

Comment:
Why is this header selected to host this expression?

Reply:
It should be in a header because it is used in two places. Any suggestions where to put it?

Reply:
I meant "why in _this_ header" rather "why in header" :)
This header looks good to me. Another possible one is `validation.h`.

Reply:
But your variant is better for a code reader, I think.